gke/standard/zonal/secondary-boot-disk/main.tf

```diff
@@ -45,3 +45,34 @@ resource "google_container_node_pool" "default" {
   }
 }
 # [END gke_standard_zonal_secondary_boot_disk]
+
+# [START gke_standard_zonal_secondary_boot_disk_data]
+resource "google_container_cluster" "default" {
```

Comment:
I think we can re-use the same cluster for both examples.  You can use a dedicated `[START gke_standard_zonal_secondary_boot_disk_cluster]` so it can be used with either node pool.

Reply:
Changed to reuse cluster